<launch>

        <rosparam>
            transform_cloud/use: false
            transform_cloud/frame_id: "base_link"
        </rosparam>

	<include file="$(find pr2_playpen)/launch/playpen_kinect.launch"/>
	<!--	<include file="$(find pr2_playpen)/launch/pt_cloud_tf_test.launch"/> -->
	<!--    <node name="reconfig_kinect_playpen" pkg="dynamic_reconfigure" type="dynparam" args="load kinect_playpen $(find pr2_playpen)/launch/run_kinect_params.yaml" />-->

	<node pkg="UI_segment_object" name="playpen_segment_cloud" type="segment_object" respawn="true" output="screen">
	      <remap from="/camera/rgb/image_color" to="/playpen_kinect/rgb/image_color" />
	      <remap from="/camera/rgb/points" to="/playpen_kinect/rgb/points" />
	      <remap from="get_object_on_plane" to="playpen_get_object" />
	      <remap from="get_3D_pt" to="playpen_get_3D_pt" />
	      <remap from="UI_reset" to="playpen_UI_reset" />
	      <remap from="save_image" to="playpen_save_image" />
	      <remap from="segment_plane" to="playpen_segment_region" />
	      <remap from="segment_object" to="playpen_segment_object" />
	      <remap from="save_pt_cloud" to="playpen_save_pt_cloud" />
	      <remap from="reset_plane_coeff" to="playpen_reset_plane_coeff" />
	</node>

	<node pkg="pr2_playpen" name="playpen_segment_cloud_pub" type="segment_client.py" output="screen" args="--node playpen_segment_cloud_pub --serv playpen_get_object" />


	
</launch>
